snuba/cli/bootstrap.py

```diff
@@ -100,11 +101,25 @@ def bootstrap(
     # tables or distributed tables, etc.
 
     # Create the tables for every dataset.
+    clickhouse_tables = ClickhousePool().execute("show tables")
```

Comment:
Is there a reason why we create a new pool/connection for each statement in here?

Reply:
I don't think it is needed. Just did the same as it was before. Will fix.